buildSrc/src/main/groovy/com/github/okbuilds/okbuck/generator/BuckFileGenerator.groovy

```diff
@@ -33,7 +35,7 @@ final class BuckFileGenerator {
         } as Map<Project, BUCKFile>
     }
 
-    private static List<BuckRule> createRules(Project project) {
+    private static List<BuckRule> createRules(Map<String, String> projectConfigTargets, Project project) {
```

Comment:
just do this at the last level. no need to pass this map around you already have project ref